streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -960,34 +990,14 @@ private void removeStandbyTasks() {
     }
 
     private void removeSuspendedTasks() {
-        log.info("{} Removing all suspended tasks [{}]", logPrefix, suspendedTasks.keySet());
+        log.info("{} Removing all suspended tasks [{}]", logPrefix, suspendedActiveTasks.keySet());
```

Comment:
See my comments above. If that case should we be guaranteed that `suspendedActiveTasks` is always an empty map here?